agr_api/src/main/java/org/alliancegenome/api/controller/OrthologyController.java

```diff
@@ -18,27 +15,25 @@
 import org.alliancegenome.neo4j.repository.OrthologousRepository;
 import org.alliancegenome.neo4j.view.HomologView;
 
+import jakarta.enterprise.context.RequestScoped;
+import jakarta.inject.Inject;
+
 @RequestScoped
 public class OrthologyController implements OrthologyRESTInterface {
 
 	public static final String API_VERSION = "0.91";
 
-	//@Context
-	//private HttpRequest request;
+	// @Context
+	// private HttpRequest request;
 
 	@Inject OrthologyCacheRepository service;
 
 	@Inject GeneController controller;
-	
+
 	private static OrthologousRepository orthoRepo = new OrthologousRepository();
 
 	@Override
-	public JsonResultResponse<HomologView> getDoubleSpeciesOrthology(String taxonIDOne,
```

Comment:
Why are you changing this? I like to have the various arguments underneath each other for ease of readability.

Reply:
This is auto formatting in eclipse.